TranslatorReasonerAPI.yaml

```diff
@@ -422,3 +449,199 @@ components:
       description: A Compact URI
       externalDocs:
         url: https://www.w3.org/TR/2010/NOTE-curie-20101216/
+    KnowledgeMap:
+      type: object
+      properties:
+        nodes:
+          type: object
+          description: >-
+            List of semantic types provided by this knowledge source.
+          additionalProperties:
+            $ref: '#/components/schemas/KMNode'
+        predicates:
```

Comment:
```suggestion
        edges:
```

Reply:
Since we are talking about using knowledge map to represent metadata for KGX as well, maybe we should use categories/predicates rather then nodes/edges. I agree that, as it is now, nodes/predicates does not make sense.

Reply:
We settled on node/edge terminology